docs/why.qmd

```diff
@@ -2,24 +2,224 @@
 title: "Why Ibis?"
 ---
 
+Ibis defines a Python dataframe API that supports any query engine -- the
```

Comment:
nitpick: let's use a more precise word than "support" for people to resonate. maybe "connect," "execute," or "query."